modules/apps/object/object-service/src/main/java/com/liferay/object/internal/search/spi/model/index/contributor/ObjectEntryModelDocumentContributor.java

```diff
@@ -151,8 +151,11 @@ private void _contribute(Document document, ObjectEntry objectEntry)
 			sb.setIndex(sb.index() - 1);
 		}
 
+		document.add(
+			new Field(
+				Field.getSortableFieldName(Field.ENTRY_CLASS_PK),
+				document.get(Field.ENTRY_CLASS_PK)));
```

Comment:
I believe we should put this new code before `Map<String, Serializable> values = _objectEntryLocalService.getValues(..)` (**line 137**) and keep the line break, since the `document.add(new Field("objectEntryContent", sb.toString()));` depends on `StringBundler sb.`

Reply:
It makes sense. I'll do that change. Thanks!